lightsaber/src/test/kotlin/schwarz/it/lightsaber/checkers/UnusedDependenciesKtTest.kt

```diff
@@ -65,6 +75,30 @@ class UnusedDependenciesKtTest {
         compilation.assertNoFindings()
     }
 
+    @Test
+    fun secondDependencyUnusedOnComponent() {
+        val component = createSource(
+            """
+                package test;
+
+                import dagger.Component;
+
+                @Component(dependencies = {Dependency.class, Dependency2.class})
```

Comment:
```suggestion
                @Component(dependencies = {
                    Dependency.class, 
                    Dependency2.class
                })
```

We would need test for things like this:

Reply:
Done.